src/ndn/transport/stream_socket.py

```diff
@@ -100,3 +101,65 @@ def __init__(self, host: str = '', port: int = 0):
     async def open(self):
         self.reader, self.writer = await aio.open_connection(self.host, self.port)
         self.running = True
+
+
+class UdpFace(Face):
```

Comment:
UDP is not a *stream* socket, so that it does not belong in `stream_socket.py`.

* Move the `Face` type to a new `face.py` file.
* Create a separate file for `UdpFace` type.